src/pages/create-manifest/create-manifest.style.js

```diff
@@ -74,6 +72,18 @@ export const fileInputStyle = css`
   display: none;
 `;
 
+export const actionPanelStyle = css`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  width: 100%;
+  background-color: #ffffff;
+  box-shadow: rgba(0, 0, 0, 0.14) 0px 2px 4px;
+  padding: 12px;
```

Comment:
generally almost everywhere we have rems here. Which I'm not a big fan of tbh, but just kept doing it cause it was started before I joined. Won't change much most likely, but make sure this discrepancy doesn't cause any distortions, especially with different zoom values

Reply:
True. I will convert it to rem

Reply:
Fixed in https://github.com/Azure/azure-maps-creator-onboarding-tool/pull/104